### PR TITLE
Update release instructions

### PR DESCRIPTION
**Pull request by blairconrad**
I noticed a few things that were a little off while working on #665. Figured I might as well get them out of the way first.

### CODE REVIEW

File: rakefile.rb [hunk 1]
```diff
@@ -44,22 +44,19 @@
...
 - [ ] tweet, mentioning contributors and post link as comment here for easy retweeting ;-)
 - [ ] post tweet in [Gitter](https://gitter.im/FakeItEasy/FakeItEasy)
 - [ ] post links to the NuGet and GitHub release in each issue in this milestone, with thanks to contributors
-- [ ] use `rake set_version[new_version]` to
-    - create a new branch
-    - change CommonAssemblyInfo.cs to expected minor version (of form _xx.yy.zz_)
-    - push to origin
-    - create PR to upstream master
+- [ ] use `rake set_version[new_version]` to create a pull request that changes the version in 
+       CommonAssemblyInfo.cs to the expected version (of form _xx.yy.zz_), and once that PR has been merged
```
Comment (adamralph) [inline rakefile.rb]: > , and once that PR has been merged

why do we have to wait for the PR to be merged?

**Comment by adamralph** [comment]
@blairconrad LGTM, I just left one question.

File: rakefile.rb (hunk 1 above)
Comment (blairconrad) [inline rakefile.rb]: That's a fair point. The real thing we need is to call `rake create_milestone` from a working copy that has the desired version number in it. So merging is not necessary. I will amend. Thanks.

**Comment by blairconrad** [comment]
Thanks, @adamralph.

**Pull request merged**